test/load/fleet-autoscaling/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM hopsoft/graphite-statsd
+
+# Locust
+RUN pip install locustio
+EXPOSE 8089 5557 5558
+
+RUN mkdir /etc/service/locust
```

Comment:
nit: /etc is typically for configuration files, not for executables

how about /opt/locust/ ?

Reply:
That won't work. The image that I am using in this Dockerfile  (https://hub.docker.com/r/hopsoft/graphite-statsd) is built from phusion which requires specific configuration when a new daemon is added. This includes an executable that runs the daemon to the /etc/service/ directory: https://github.com/phusion/baseimage-docker#adding-additional-daemons